<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestorePage</class>
 <widget class="QWidget" name="RestorePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>数据恢复</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>仓库:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="repositoryComboBox">
       <property name="minimumWidth">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="snapshotTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>快照ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>创建时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主机名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>路径</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>标签</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="restoreOptionsGroup">
     <property name="title">
      <string>恢复选项</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_options">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_target">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>恢复到:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetPathEdit">
          <property name="placeholderText">
           <string>选择恢复目标路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>浏览...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_options">
        <item>
         <widget class="QCheckBox" name="includeCheckBox">
          <property name="text">
           <string>包含特定文件/目录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="includeEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>输入包含路径（可选）</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_buttons">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="restoreButton">
          <property name="text">
           <string>开始恢复</string>
          </property>
          <property name="minimumWidth">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
